docs/how_to_use/how_to_build.md

```diff
@@ -227,6 +227,14 @@ For example, add the following to `.bazelrc.user` to build rust on linux:
 build --build_tag_filters -objc,-cuda
 ```
 
+### Build Rust code
+
+The current toolchain is using the nightly channel, the following build flag is needed, add this to your `.bazelrc.user`
+
```

Comment:
Since the current toolchain is using the nightly channel, the following build flag is needed. Add this to your `.bazelrc.user`.